### ISSUE TITLE
Fail to get the result from `FZF_DEFAULT_COMMAND` in Vim on Windows

### ISSUE THREAD

**Issue by wontoncc** [junegunn/fzf#847]
- Category
    - [ ] fzf binary
    - [ ] fzf-tmux script
    - [ ] Key bindings
    - [ ] Completion
    - [x] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [ ] Linux
    - [ ] Mac OS X
    - [x] Windows
    - [ ] Windows Subsystem for Linux
- Shell
    - [ ] bash
    - [ ] zsh
    - [ ] fish
    - [x] cmd

I tried to use `fzf` and `fzf.vim` with `FZF_DEFAULT_COMMAND` on Windows. When running `:Files` a `cmd` window pops up with some welcome text for `cmd` on it but no result.
After some digging into the `fzf.vim` I found the plugin executes the `FZF_DEFAULT_COMMAND` in the temporary file (.tmp) with `cmd`, which leads to the welcome text in the output but not to actually execute the command.

With `cmd /c` being able to execute the command in a batch file, I tried to make a work-around:
```vim
" plugin/fzf.vim

  if !has_key(dict, 'source') && !empty($FZF_DEFAULT_COMMAND)
    let temps.source = tempname()
    call writefile(split($FZF_DEFAULT_COMMAND, "\n"), temps.source . (s:is_win ? '.bat' : ''))
    let dict.source = (empty($SHELL) ? &shell : $SHELL) . (s:is_win ? ' /c' : '') . ' ' . s:shellesc(temps.source)
  endif
```

And it requires the `FZF_DEFAULT_COMMAND` starts with '@' to leave out the command line in the list of result.

**Comment by junegunn** [comment]
Thank you, I applied your patch. Please let me know if you see any other issues.

**Comment by junegunn** [comment]
> And it requires the FZF_DEFAULT_COMMAND starts with '@' to leave out the command line in the list of result.

Do you think we should prepend `@` to the command automatically?

**Comment by wontoncc** [comment]
My bad, that should be implemented in the patch.
To achieve that might involve adding `@` to every line of the commands. Better add `@echo off` to the head of the temporary file in my opinion.

```vim
call writefile(['@echo off'] + split($FZF_DEFAULT_COMMAND, "\n"), temps.source . (s:is_win ? '.bat' : ''))
```

**Comment by junegunn** [comment]
I see, thanks for the update. Fixed in 	c2db67c.
